### PR TITLE
Update AppGitStatusPanel.tsx

### LINKED ISSUE #3585: 🐛 Bug Report: Last sync value is incorrect
### What happened?

When you sync to GitHub and look at the Last sync value in the client, it's not right.

Screen Shot 2022-08-25 at 10 57 49

This happened before the first build
### What you expected to happen

Show the correct time

### How to reproduce

Generate a resource and connect to GitHub

### Amplication version

0.15.0

### Environment

_No response_

### Are you willing to submit PR?

Yes I am willing to submit a PR!

### PR DESCRIPTION
**Pull request by prakhar-198**
Close #3585 

## PR Details

displayed "never" instead of date.

## PR Checklist
- [ ] Tests for the changes have been added
- [ ] `npm test` doesn't throw any error

IMPORTANT: Please review the [CONTRIBUTING.md](https://github.com/amplication/amplication/blob/master/CODE_OF_CONDUCT.md) file for detailed contributing guidelines.

### CODE REVIEW

**Review by TheLearneer — CHANGES_REQUESTED** [review]
That doesn't seem to be correct 😓 
You will still have to show the last sync date if it's available.

File: packages/amplication-client/src/Resource/git/AppGitStatusPanel.tsx [hunk 1]
```diff
@@ -75,10 +75,9 @@ const AppGitStatusPanel = ({ resource, showDisconnectedMessage }: Props) => {
             <div className={`${CLASS_NAME}__last-sync`}>
               <Icon icon="clock" />
               <Tooltip
-                aria-label={`Last sync: ${format(lastSync, DATE_FORMAT)}`}
+                aria-label={`Last sync: never`}
```
Comment (abrl91) [inline packages/amplication-client/src/Resource/git/AppGitStatusPanel.tsx:78]: 1. Why did you remove the `format` method?
2. please don't use hard-coded "never". It is "never" only when there is no connection to GitHub, but if the user did sync with GitHub, the last sync should be a valid date with a specific format.

Comment (prakhar-198) [reply packages/amplication-client/src/Resource/git/AppGitStatusPanel.tsx:78]: Fine..I'm on it.

**Comment by abrl91** [comment]
@prakhar-198 any updates?

**Comment by abrl91** [comment]
https://github.com/amplication/amplication/pull/4272
